SearchQnA/docker_compose/amd/gpu/rocm/compose.yaml

```diff
@@ -0,0 +1,168 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  search-tei-embedding-service:
+    image: ghcr.io/huggingface/text-embeddings-inference:cpu-1.5
+    container_name: search-tei-embedding-server
+    ports:
+      - "3001:80"
+    volumes:
+      - "/var/lib/GenAI/search/data:/data"
```

Comment:
The volume here should be './data' by default, since we don't offer default data folder.

Reply:
It's been fixed